packages/client-core/src/common/services/FileBrowserService.ts

```diff
@@ -67,7 +67,7 @@ export const FileBrowserService = {
       fileBrowserState.merge({
         files: files.data,
         skip: files.skip,
-        total: files.total,
+        total: files.data.length,
```

Comment:
is this correct? shouldn't `total` be total count of items, not just however many is returned in our pagination?

Reply:
Updated